ebos/eclproblem.hh

```diff
@@ -2522,7 +2522,7 @@ private:
             comm.broadcast(&size, 1, 0);
             comm.broadcast(buffer.data(), pos, 0);
         } else {
-            int size;
+            std::size_t size;
```

Comment:
this is correct, but i'd rater send position above, ie, send the int.

Reply:
Please do that.